### PR TITLE
Adding api methods for suggestions and suggestions/:slug

### PR DESCRIPTION
**Pull request by trentstollery**
Hey!

So I needed these endpoints for a project I'm working on at work so I thought I would add them :)
Follows your normal layout and I added some classes too.
Works like a charm :)
Happy to clean up/Add any more doco so we can merge it back in just let me know.
It's been kind of a rushed job since I needed it to work asap haha.
Thanks for your lib it rocks!!

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ def read(*paths):
 
 setup(
     name='python-twitter',
-    version='2.3',
+    version='2.3.1',
```
Comment (trentstollery) [inline setup.py]: This is just my idea for the version jump... not sure if you use this sort of versioning but it maybe sane for such a small jump like this

**Comment by bear** [comment]
we do use patch levels, but the 2.3 is already a jump for a larger change -- so no need to bump it IMO

**Comment by trentstollery** [comment]
I agree with the version issue, I've swapped this back to 2.3 :)

**Comment by trentstollery** [comment]
I also required the media data from a tweet. So I've added a commit to add that to a class :).
I think you where half way to doing that but I thought I would finish it up!

**Comment by bear** [comment]
I need to fix the travis config to not test on python v3 until the python v3 branch is live...

long winded way of saying that i'm going to merge this - thanks for the PR!

**Pull request merged**